BUILDING.md

```diff
@@ -311,8 +311,8 @@ $ make
 
 ### `Intl` (ECMA-402) support:
 
-[Intl](https://github.com/nodejs/node/wiki/Intl) support is
-enabled by default, with English data only.
+[Intl](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl)
```

Comment:
Alternatively we could link to https://github.com/nodejs/node/blob/master/doc/api/intl.md instead as ECMA-402 refers to more than just the global Intl object?